Sources/Atomics/atomics-reference.swift

```diff
@@ -120,14 +108,10 @@ extension AtomicReference
   @inlinable
   public mutating func load(order: LoadMemoryOrder = .sequential) -> T?
   {
-    if let pointer = ptr.spinLoad(.lock, order)
+    if let pointer = ptr.load(order)
     {
-      assert(ptr.load(.sequential) == UnsafeRawPointer(bitPattern: 0x7))
```

Comment:
this assert is a sanity check to make sure the atomic variable was changed to the "locked" state.